metron-platform/metron-enrichment/src/main/java/org/apache/metron/enrichment/writer/SimpleHbaseEnrichmentWriter.java

```diff
@@ -169,16 +188,19 @@ public HTableInterface getTable(Map<String, Object> config) throws IOException {
   private List<String> getColumns(Object keyColumnsObj, boolean allowNull) {
     Object o = keyColumnsObj;
     if(allowNull && keyColumnsObj == null) {
+      LOG.debug("No key columns were specified");
       return Collections.emptyList();
     }
     if(o instanceof String) {
+      LOG.debug("Key column: '" + o + "'");
```

Comment:
Better to turn this and similar statements into `LOG.debug("Key column: '{}'", o );`. Per the slf4j docs, this has the potential to improve calls by up to 30x where debugging is not enabled.